### PR TITLE
Pull request for sileht/ceph-log-hide

### PR DESCRIPTION
**Pull request by sileht**
test: don't print the whole config file
tests: hide useless rados output

### CODE REVIEW

File: gnocchi/tests/base.py [hunk 1]
```diff
@@ -337,7 +337,8 @@ def setUp(self):
         elif self.conf.storage.driver == 'ceph':
             pool_name = uuid.uuid4().hex
             subprocess.call("rados -c %s mkpool %s" % (
-                os.getenv("CEPH_CONF"), pool_name), shell=True)
+                os.getenv("CEPH_CONF"), pool_name), shell=True,
+                stdout=open(os.devnull, 'w'), stderr=subprocess.STDOUT)
```
Comment (jd) [inline gnocchi/tests/base.py]: this looks like it'll never be closed, so I'd go `subprocess.PIPE`, no?

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 52229a8e9fcb66d75c479acb09734ce6284c519e

File: gnocchi/tests/base.py (hunk 1 above)
Comment (sileht) [inline gnocchi/tests/base.py]: subprocess.PIPE can hang if don't read the pipe

File: gnocchi/tests/base.py [hunk 2]
```diff
@@ -336,8 +336,10 @@ def setUp(self):
                                    'storage')
         elif self.conf.storage.driver == 'ceph':
             pool_name = uuid.uuid4().hex
-            subprocess.call("rados -c %s mkpool %s" % (
-                os.getenv("CEPH_CONF"), pool_name), shell=True)
+            with open(os.devnull) as f:
```
Comment (chungg) [inline gnocchi/tests/base.py]: this defaults to 'r'... it should be 'w'?

**Comment by sileht** [comment]
Pull-request updated, HEAD is now edd394460992c04cfff243dd9e04c53996250569

**Pull request merged**